Enhance Edge3 Provider docs (#49859)
mistercrunch/Airflow: providers/edge3/docs/deployment.rst

Worker maintenance can also be triggered via the CLI command on the machine that runs the worker.

.. code-block:: bash

    airflow edge maintenance --comments "Some comments for the maintenance" on

This will stop the local worker instance from accepting new tasks and will complete running tasks.
If you add the command argument ``--wait`` the CLI will wait until all
running tasks are completed before return.

If you want to know the status of you local worker while waiting on maintenance you can
use the command

.. code-block:: bash

    airflow edge status

This will show the status of the local worker instance as JSON and the tasks running on it.

The status and maintenance comments will also be shown in the web UI
in the "Admin" - "Edge Worker Hosts" page.

.. image:: img/worker_maintenance.png

The local worker instance can be started to fetch new tasks via the command

.. code-block:: bash

    airflow edge maintenance off

This will start the worker again and it will start accepting tasks again.

.. _deployment:maintenance-mgmt-cli:

Worker Maintenance Management CLI
---------------------------------

> Besides the CLI command to trigger maintenance on the local worker instance, there is also a command to

Reviewer: ```suggestion
Besides the CLI command to trigger maintenance on the local worker instance, there are also additional commands to
```

> manage the maintenance of all workers in the cluster. This command can be used to trigger maintenance

Reviewer: ```suggestion
manage the maintenance of all workers in the cluster. These commands can be used to trigger maintenance
```

on all workers in the cluster or to check the status of all workers in the cluster.

> This set commands need database access, such can only be called on the central Airflow

Reviewer: ```suggestion
These set of commands need database access, and can only be called on the central Airflow
```
